pyqtgraph/graphicsItems/PlotDataItem.py

```diff
@@ -569,14 +578,41 @@ def getData(self):
                         x = x[x0:x1]
                         y = y[x0:x1]
                     
-            if ds > 1:
+        
+            if len(x) > 1 and ds > 1:
+                n = len(x) // ds
+                if self.opts['quantizedDownsample']:
+                    i0 = np.searchsorted( x, ds*(1 + x[0]//ds ))
+                    if i0 < x[-1]:
+                        indices = np.arange(i0, n*ds, ds)
+                    else:
+                        indices = [0]
+                    """use midpoint of x downsample points"""
+                    x = x[indices] + float(x[-1]-x[0]) / (len(x)-1)
```

Comment:
I guess it could be an option, but (especially with the mean) I think it makes more sense that a value calculated over a bucket interval should be plotted at the bucket's midpoint